src/globals.js

```diff
@@ -18,3 +19,7 @@ doc.implementation.createHTMLDocument = function( html ) {
 
 // `hpq` depends on `document` be available globally
 global.document = doc;
+
+if ( ! global.window.Node ) {
```

Comment:
Ah! question, does defining this in this way fix the red screen I got saying `undefined is not an object (evaluating '_window$Node.TEXT_NODE')`?

Reply:
Yes! So, turns out that recent Gutenberg changes exposed the need for those constants from `window.Node` that were previously handled from `dom-react` and our `jsdom-core` dep. But yeah, with the changes we apparently need to manually add the support.

Reply:
excellent, thanks for this clarification!